src/Configuration/Programmer/Programmer_helper.cpp

```diff
@@ -167,13 +197,19 @@ std::vector<TapInfo> extractTapInfoList(const std::string& tapInfoString) {
     if (tokens.size() == 8) {
       TapInfo tapInfo;
       tapInfo.index = ConvertIntegerStringToInt(tokens[0]);
+      tapInfo.index = ConvertIntegerStringToInt(tokens[0]);
```

Comment:
Why need double-coding? Line 199 vs Line 200

Reply:
It was mistake.